client/post-editor/post-editor.jsx

```diff
@@ -431,19 +492,21 @@ export const PostEditor = React.createClass( {
 	},
 
 	restoreAutosave: function() {
-		var edits,
-			autosaveData = this.state.post.meta.data.autosave;
-
-		this.setState( { showAutosaveDialog: false, isLoadingAutosave: true } );
-
-		edits = {
-			title: autosaveData.title,
-			excerpt: autosaveData.excerpt,
-			content: autosaveData.content
-		};
+		this.setState( { showAutosaveDialog: false } );
+		this.restoreRevision( this.state.post.meta.data.autosave );
```

Comment:
Though this access isn't new to this PR, how sure are we that this nested property can be safely accessed? Does it warrant the use of `get`?